tools/slang-test/slang-test-main.cpp

```diff
@@ -1004,7 +1004,10 @@ TestResult runSimpleTest(TestContext* context, TestInput& input)
     CommandLine cmdLine;
     _initSlangCompiler(context, cmdLine);
 
-    cmdLine.addArg(input.filePath);
+    if (input.testOptions->command != "SIMPLE_EX")
```

Comment:
At some point we are going to need to turn this all around and have the test files be the canonical source on what the command-line should be, and just let them use substitutions like `${slangc}` or `${api}` as markers that the test runner fills in with paths/options/whatever. We have a whole lot of test types now that just amount to slight tweaks in what the automatically-inserted arguments are.

One other nice thing that could arise if we have the tests themselves drive the command lines is that we can start to think about tests that require multiple command lines (multiple invocations of `slangc` or other tools), which seems necessary for debugging stuff like this repro feature or any eventual container serialization feature.

Reply:
Yes - this is getting a bit out of hand.